webpack.config.babel.js

```diff
@@ -178,6 +178,7 @@ module.exports = ({ stats = false } = {}) => ({
       // AUTOCONNECT: only applies for mock implementation
       AUTOCONNECT: 'true',
       PRICE_ESTIMATOR_URL: process.env.PRICE_ESTIMATOR_URL || 'develop',
+      APP_ID: process.env.APP_ID,
```

Comment:
I think that if you are not setting a default value for `APP_ID`, there's no need to have this line.
`process.env.INFURA_ID` for instance is used else where and it's not here.

```suggestion
```

Reply:
Not how it works. EnvironmentPlugin never iterates over `process.env` keys
https://github.com/webpack/webpack/blob/master/lib/EnvironmentPlugin.js#L16-L24
You can try console.logging it to make sure

So `INFURA_ID` was broken all this time? Oops :no_mouth: 
Maybe other envs too